README.md

```diff
@@ -81,14 +81,17 @@ target is a url (http load tests) or host:port (grpc health test).  flags are:
 	grpc server port. Can be in the form of host:port, ip:port or port.
 	(default "8079")
   -cert
-	Full path to the certificate used for grpc TLS. Applicable to Fortio
-	load, grpcping, and server commands. Preface the grpc destination with
-	"https://" if standard TLS (i.e. valid server certificate signed by a
-	trusted CA) is desired. -cert will override "https://" if both are provided.
-	(default "") means no user-provided cert.
+	Full path to the certificate used for grpc TLS. Applicable to the server command.
+	(default "") means no user-provided certificate.
   -key
-	Full path to the key required for grpc TLS. Applicable to Fortio server.
+	Full path to the key required for grpc TLS. Applicable to the server command.
 	(default "") means no user-provided key.
+  -ca-cert
```

Comment:
sorry to be a PITA but please use  cacert not ca-cert, trying to be command line compatible with curl's:
```
    --cacert FILE   CA certificate to verify peer against (SSL)
```

Reply:
( mentioned a few times, starting on https://github.com/istio/fortio/pull/204#pullrequestreview-107094841 )